k8s/spinnaker/README.md

```diff
@@ -10,97 +10,112 @@ If you would like to learn more about Spinnaker, please visit the
 
```

Comment:
Where does this readme appear? Does this show up in the Marketplace with the solution, or just in the click-to-deploy repo? Reason I ask is I'm wondering why we have a sentence, and a level-2 heading, describing Google Click to Deploy if this readme is for Spinnaker specifically.

Reply:
should match other click-to-deploy k8s apps

Reply:
Hm, it looks like what's in the text files may actually not be what's in the Marketplace UI, see e.g. https://github.com/duftler/click-to-deploy/blob/master/k8s/nginx/README.md#about-google-click-to-deploy

Will leave as-is, in that case.